pkg/hashtable/hashtable.go

```diff
@@ -0,0 +1,63 @@
+package hashtable
+
+import (
+	"github.com/neelkshah/pandora/config"
+	"github.com/neelkshah/pandora/pkg/helper"
+	ll "github.com/neelkshah/pandora/pkg/linkedlist"
+)
+
+type HashTable struct {
+	table   []*ll.LinkedList
+	buckets uint32
```

Comment:
What is the buckets variable meant for?

Reply:
number of buckets/length of table. Can you suggest a better name or analogy to follow for consistent naming?

Reply:
I think the type should be `int` here. It's something we're using for counting. As far as naming goes, maybe `numberOfBuckets`? Although I don't really see an issue here.

Reply:
Same for inserts and deletes. Just so we can avoid unnecessary type conversions for looping and comparisons.